### ISSUE TITLE
friend request notification not stable 

### ISSUE THREAD

**Issue by SalmanKhan86** [moxi9/phpfox#196]
i received friend request and accept it but its still show count one in notification window.
![friend request notification bug](https://cloud.githubusercontent.com/assets/13433124/8843154/792da3a8-3122-11e5-88d1-523abfaa55f3.png)

**Comment by Jinkers** [comment]
This has already been mentioned before in issue #41. Unsure of what @RaymondBenc and @Data66PF have done about this, as the issue was closed and opened multiple times. A simple refresh (sometimes it needs a control/command+f5) should remove the notification.

**Comment by SalmanKhan86** [comment]
This Bug is not fixed yet.solution (ctrl+f5) given by @Data66PF in previous thread is not acceptable.Thanks @Jinkers for adding this to previous Thread.

**Comment by Data66PF** [comment]
This is a duplicate of the open one. Note that having multiple issues in a ticket results in the issue you saw in the other ticket. he has to close it to put the bug as closed for each issue. so it's open, closed, open, closed. I'll leave this one open and close that one instead. It takes a page refresh to do things on the site. I believe this is how it's designed.

**Comment by SalmanKhan86** [comment]
sorry @Data66PF by mistake i posted here i haven't aware of same thread already there.

**Comment by Data66PF** [comment]
It's ok @SalmanKhan86 , The other one was confusing with too many things in it. That's why it's better to put only one issue per bug as it gets hard to follow. The only exception to that though would be if one of us asked you to post them all in one, such as this one, #64 , if you find settings that need to be enabled (like this one), it's also ok to have posted it in that bug. :)

**Comment by SalmanKhan86** [comment]
Thanks a lot @Data66PF

**Comment by Data66PF** [comment]
No problem. Trying to make sure everyone is a "happy camper" ;)

**Comment by Data66PF** [comment]
Since it appears designed like this, I'll label it an enhancement in order to look at any way to adjust it.
